app/views/layouts/_main_nav.html.erb

```diff
@@ -1,7 +1,7 @@
 <% default_nav_links = NavigationLink.default_section.ordered.to_a %>
 <% other_nav_links = NavigationLink.other_section.ordered.to_a %>
 
-<nav class="mb-4 <% unless user_signed_in? %>mt-4<% end %>" id="main-navigation" aria-label="<%= community_name %>">
+<nav class="mb-4 <% unless user_signed_in? %>mt-4<% end %> main-navigation" aria-label="<%= community_name %>">
```

Comment:
Are the `main-navigation` and `other-nav` classes only used for the purpose of checking they exist in the specs? If so, I think it would be an improvement to remove the classes, and instead use e.g. `data-testid="main-nav"`, which we could then find in the specs via the selector `nav[data-testid="main-nav"]`

Reply:
@aitchiss yes they are. I thought classes were straightforward but using data elements instead might prevent future collisions.